### ISSUE TITLE
Restore from checkpoint fails if messages.log is removed

### ISSUE THREAD

**Issue by tjwatson** [OpenLiberty/open-liberty#18395]
If a server is restored from a checkpoint after the logs/ directory has been cleared then the criu restore operation will fail with the following in the restore.log:

```
(00.007538) 203711: Error (criu/files-reg.c:1767): Can't open file .../wlp/usr/servers/petclinic/logs/messages.log: No such file or directory
(00.007542) 203711: Error (criu/files.c:1210): Unable to open fd=13 id=0x39
(00.007622) Error (criu/cr-restore.c:2397): Restoring FAILED.
(00.007816) Error (criu/cr-restore.c:1479): 203711 killed by signal 9: Killed

```

A checkpoint hook is needed to close/reopen the messages.log on checkpoint/restore to avoid checkpointing the file handle for messages.log in the checkpoint image.

**Comment by tjwatson** [comment]
We actually should force the messages and trace logs to rotate on restore to start with a fresh log each restore.

**Comment by tjwatson** [comment]
This has been fixed a long time ago with PR #19339
